### PR TITLE
Golden apples mutate into poison apples, which have a new chem

### PR DESCRIPTION
**Pull request by Buggy123**
alternative to #35820
closes #35820

Having a unused item in the game is bad, but I thought I'd make a alternative PR that addresses the other possible solution (remove vs improve).

I have not played botany at all in a very long time this might be insanely overpowered, or not. No clue.

🆑 
tweak: Nanotrasen has... uh... I don't know, whatever. Golden apples can now mutate into poison apples.
IC changelogs are nice but I got 2 hours of sleep so fuck that.
/🆑 

Kindleium is a new reagent. On entering your system (and with a 5% chance every tick you _do not_ have the status effect) it effectively mirrors the effects of getting hit by a kindle bolt. Notable differences are that null rods don't nullify it, and it can effect clock cultists.

I'd turn it into a universal proc instead of copypasting but I don't know where to _put_ it.

### CODE REVIEW

**Comment by vuonojenmustaturska** [comment]
We don't actually want to put zombie powder in the hands of botany

**Comment by Buggy123** [comment]
I have basically never played botany in its current state and have no idea what is or isn't balanced.

**Comment by vuonojenmustaturska** [comment]
zombie powder is what's in the sleepy pen

**Comment by ChangelingRain** [comment]
what part of "really, really not something we'd want botany able to easily access" did you not understand?

**Comment by Buggy123** [comment]
No I know what it is, and what it does. It's a super-stun chem which puts you to sleep the instant it enters your system, metabolizes slowly, makes you appear to be dead(and thus draws less suspicion), and is mildly toxic. Shit's hilarious if you emag the chem dispenser and then load the smoke machine with it.

But in botany, I have no idea if getting this mutation takes 3 minutes or 30 minutes or 3000 minutes.

**Comment by Repukan** [comment]
No, trying to punish a feature after mantainers say they don't want it in, is the real bad idea.

**Comment by vuonojenmustaturska** [comment]
@Buggy123 all mutations are effectively roundstart, the only limiters are things like cannabis that don't always mutate into the thing you want and have a limited amount of seeds, two problems apples don't and won't have

**Comment by Buggy123** [comment]
Wait, Changelingrain is a maintainer?

Either way maintainer opinions differ massively so just because 1 maintainer hates this idea doesn't mean that others don't. I mean, have you _seen_ Goofball?

**Comment by Buggy123** [comment]
>@Buggy123 all mutations are effectively roundstart, the only limiters are things like cannabis that don't always mutate into the thing you want and have a limited amount of seeds, two problems apples don't and won't have

Oooooooohhhh...

Uh.

**_Well_**, might be wise to close this then.

I don't suppose decreasing the mutation chance would help at all? That seems to be a thing you can do.

**Comment by RandomMarine** [comment]
Why not just give it a different toxin?

**Comment by 81Denton** [comment]
I'm all for bringing back unused assets, but zombie powder is absolute nuts and makes all other KO chems look like a joke.

@vuonojenmustaturska Sleepypens have a snowflake version of chloral that kicks in later, but sleeps people for longer. Zombie powder is an instant KO + fake death

**Comment by Buggy123** [comment]
That's not a bad idea. The original toxin appears to have been cyanide so...

**Comment by Buggy123** [comment]
see title

**Comment by 81Denton** [comment]
Looks solid to me. I don't see an issue balance wise since cyanide is mediocre compared to coniine/other toxins and can already be obtained through deathweed.

**Comment by Buggy123** [comment]
Would there be another poison which is better? I thought about initropidril because suddenly having a heart attack and keeling over sorta fits with the sleeping beauty theme but that might be too strong.

**Comment by vuonojenmustaturska** [comment]
@Buggy123 the way mutations in botany work is that you get unstable mutagen from the chemist and apply it in 5u drops on the plant until it mutates. Each 5u drop has a 10% chance of mutating the plant to a species in the mutate list. Mutagen is a thing you should never be in short supply of, if you know what you're doing.

**Comment by 81Denton** [comment]
Initropidril can kill extremely quickly if you're unlucky and get hit with a heart attack.
Chloralhydrate2/Delayed Chloral Hydrate could work, it's essentially a gimped version of chloral that takes twice as long to kick and and doesn't deal any tox damage.

**Comment by Buggy123** [comment]
Oh, really? That hasn't changed? Don't they get a gene machine thing and strange plants with random reagents and trait things and other shit?

**Comment by vuonojenmustaturska** [comment]
yeah but the gene machine can't mutate into a different plant, it can only splice in traits and reagents from seeds you already have to seeds you already have

**Comment by 81Denton** [comment]
@Buggy123 That sounds good, it would make it something unique instead of just adding a chemical that's weaker chloral.

**Comment by arsserpentarium** [comment]
cyanide can be very easy accessable. Put there something more exotic, or add oxygen. To worth handle these apples.

**Comment by davethwave** [comment]
"During our experimentation on gene manipulation on apple mutation to gold apples a new undiscovered variety of golden apple seems to be unlocked. In addition any nanotrasen employees in possession of or growing of said apple is now a major crime."

**Comment by DaedalusGame** [comment]
selea, what are you talking about, there's no botany crop that makes cyanide and you need oxygen to make cyanide, which you by your own admission, can't get through botany

**Comment by Buggy123** [comment]
Well now it makes Kindleium instead.

**Comment by 81Denton** [comment]
@DaedalusGame Poison berries have cyanide

**Comment by DaedalusGame** [comment]
i thought they had coniine or someshit

**Comment by Buggy123** [comment]
>Given that this is gated behind 2 transformations it'd be nice if it were slightly more potent than just cyanide again, but I don't have any helpful suggestions.

I mean, it has kindleium instead of cyanide now, what do you think of that.

**Comment by 81Denton** [comment]
You should probably update the title and shorten it a little

**Comment by arsserpentarium** [comment]
I don't even know, what is kindlenium.What it supposed to do?

**Comment by Buggy123** [comment]
Its exactly like getting hit by kindle. It kindles you when it first enters your system, has a 5% chance to kindle you again at any point while its in your system and you aren't already kindled, and it removes the kindle when it exits your system.

**Comment by DaedalusGame** [comment]
aka you just gave a clockcult special to botany why

**Comment by Buggy123** [comment]
Well, for one, same reason as Narsour: plausible deniability. Kindle has a distinctive sound and distinctive slurring effect. If you hear or see anything along those lines you immediately get ";CLOCK CULT", so clock cult rounds are basically 50% steamroll, 50% discovered immediately and then sodomized for 40 minutes by a long brass rod.

Admittedly its not quite as effective as Narsour, but its a positive, however small.

**Comment by DaedalusGame** [comment]
weakness
no +1 from me

**Comment by Buggy123** [comment]
Weakness?

**Comment by JJRcop** [comment]
Can you please shorten the PR title to "Golden apples mutate into poison apples, which have a new chem"?

**Comment by Buggy123** [comment]
Okay enough title memes

**Comment by arsserpentarium** [comment]
How about neurotoxin,imprednezene,formaldehyde?

**Comment by Buggy123** [comment]
I dunno, @81Denton like kindleium, and @DaedalusGame doesn't support it apparently but won't say why. I'll wait and see what other people think.

**Comment by Buggy123** [comment]
Okay @FantasticFwoosh has given me some ideas. Plans for when I get to something that can run git/byond:

1: rename and possibly resprite to brass apples
2: replace nutriment, etc with metal
3: possibly put steam in them?

**Comment by arsserpentarium** [comment]
1.Or put assort of neurotoxins to get brainkilling apples.
For most fun,gene engineer them to add earthsblood.to negate tox and max brain dmg.
2.brass apples needs enought work to be another possible mutation of gold apples.
copper
kindlenium
iron

**Comment by improvedname** [comment]
except when botanists already can get carptoxin so zombie powder shouldnt be a issue besides its a reference lol

**Comment by arsserpentarium** [comment]
That's for refference,yeah!why do you think,that people so mean?

**Comment by ExcessiveUseOfCobblestone** [comment]
You know there's a machine literally dedicated to messing with the chemicals within a plant, right?

**Comment by arsserpentarium** [comment]
But copper needs for refference.

**Comment by Kromgar** [comment]
Botany being able to mass produce zombie powder could have 0 terrible consequences

**Comment by epochayur** [comment]
what a pointless PR

**Comment by ExcessiveUseOfCobblestone** [comment]
> Botany being able to mass produce zombie powder could have 0 terrible consequences

they already can @Kromgar ever since morphine became a producible recipe.

No one ever does it though.

File: code/modules/hydroponics/grown/apple.dm [hunk 1]
```diff
@@ -31,7 +31,7 @@
 /obj/item/seeds/apple/poisoned
 	product = /obj/item/reagent_containers/food/snacks/grown/apple/poisoned
 	mutatelist = list()
-	reagents_add = list("zombiepowder" = 0.5, "vitamin" = 0.04, "nutriment" = 0.1)
```
Comment (Iamgoofball) [inline code/modules/hydroponics/grown/apple.dm]: ok so this kills the snow white joke

**Comment by Iamgoofball** [comment]
yeah zombie powder isn't a concern anymore

plus removing zombie powder kills the joke about poison apples, did you ever see snow white

**Comment by Repukan** [comment]
I don't watch Western anime

**Comment by Buggy123** [comment]
Okay, so, i'm thinking brass apples should be a new thing. Fuck it, back to normal

**Comment by Buggy123** [comment]
Gotta wait till I get home cause I dunno how to revert commits via web interface.

**Comment by arsserpentarium** [comment]
please, add into poison apples neurotoxin and imprednezene.

**Comment by Kromgar** [comment]
@ExcessiveUseOfCobblestone How? Don't you need carpotoxin?

**Comment by arsserpentarium** [comment]
you have

**Comment by ExcessiveUseOfCobblestone** [comment]
@Kromgar Botany can make carpotoxin

**Comment by newfren** [comment]
The actual bottleneck for zombie powder for botany is actually copper. They could always get morphine from reishi.

**Comment by Kromgar** [comment]
@ExcessiveUseOfCobblestone 

How the fuck does it even work?


WIRE IS MADE OF IRON AND GLASS HOW DO YOU GET COPPER FROM WIRES MADE OF IRON AND GLASS

**Comment by arsserpentarium** [comment]
It uses alchemical transmutation.grinder changes configuration o spirits of fire to adjust influence of planets. so iron becomes copper.

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 2]
```diff
@@ -914,3 +914,34 @@
...
+	kindle(M)
+
+/datum/reagent/toxin/kindleium/on_mob_life(mob/living/M)
+	kindle(M)
+
+/datum/reagent/toxin/kindleium/on_mob_delete(mob/living/M)
+	if(M.has_status_effect(STATUS_EFFECT_KINDLE))
+		M.remove_status_effect(STATUS_EFFECT_KINDLE)
+
+/datum/reagent/toxin/kindleium/kindle(M)
```
Comment (ShizCalev) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: /proc, typecast

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 3]
```diff
@@ -914,3 +914,34 @@
 
 /datum/reagent/toxin/mimesbane/on_mob_delete(mob/living/L)
 	L.remove_trait(TRAIT_EMOTEMUTE, id)
+
+/datum/reagent/kindleium //works like if you get hit by kindle
```
Comment (ShizCalev) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: missing /toxin here

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
@Kromgar idk im not a maintainer :^)

I have to break a few more things before I get that title!

**Comment by 81Denton** [comment]
@Kromgar You can print wrapping paper from metal and glass. Spessmen have clearly developed universal constructors.

**Comment by Buggy123** [comment]
Making a new PR because this one is all over the place

**Pull request closed without merging**